api/server.go

```diff
@@ -94,9 +95,12 @@ func (s *server) consensusTipStateHandler(jc jape.Context) {
 func (s *server) syncerPeersHandler(jc jape.Context) {
 	var peers []GatewayPeer
 	for _, p := range s.s.Peers() {
-		info, ok := s.s.PeerInfo(p.Addr())
-		if !ok {
-			jc.Error(errors.New("peer not found"), http.StatusNotFound)
+		info, err := s.s.PeerInfo(p.Addr())
```

Comment:
If the PeerInfo isn't in the database, we should return the zero value anyway rather than a 404. The peer is clearly connected.